tests/test_distributed.py

```diff
@@ -97,11 +93,13 @@ def test_multitask_distributed(self):
         # Tests that DialogData.get() is doing the right thing
         # Ensure no duplication of examples among workers
         # It would be 200 if each worker did all the examples
-        self.assertEqual(valid['exs'].value(), 500)
-        self.assertEqual(test['exs'].value(), 500)
+        self.assertEqual(valid['exs'].value(), 4 * 4 + 4)
```

Comment:
should this `4` be a constant somewhere? assuming it relates to `corpus_size`

Reply:
Yes it should be.